.pre-commit-config.yaml

```diff
@@ -24,7 +24,7 @@ repos:
     rev: 'v0.0.238'
     hooks:
       - id: ruff
-        files: ^(great_expectations/core)  # TODO: expand to all of gx
+        files: ^(great_expectations|contrib) # TODO: add tests/ scripts/ etc.
```

Comment:
Now that all of `great_expectations` is being linted we can use the pre-commit hook on more than just `great_expectations/core`